### DISCUSSION TITLE
How can I delete the users database to a reset/refreshed state?

### DISCUSSION THREAD

**Discussion by oliverbob** [open-webui/open-webui#477] [General]
How can I reset the database record if I want to pull new changes and restart from scratch? The UI always tells me that the user exists (but I could not remember my admin password). How can I delete the old entries from which file?

Thanks.

**Comment by justinh-rahb** [comment]
To reset the user database you can delete the /app/backend/data/ollama.db file of the container, wherever you have the data directory mounted on the host. If you have not mounted that directory or created a persistent volume for it, your ollama.db is reset each time you repull the container image.

**Reply by ThePCMaster2026** [reply]
that works but then i have to delete the docker container and recreate it
then it works tho thanks

**Comment by oliverbob** [comment]
There are already users in this table. I guess this too needs documentation from the docs team out there. How do we change the password of a user? I don't see any password field in the table structure:

```
SELECT name FROM sqlite_master WHERE type='table';

chat
user
auth
modelfile
prompt
document
sqlite>

PRAGMA table_info(user);

0|id|VARCHAR(255)|1||0
1|name|VARCHAR(255)|1||0
2|email|VARCHAR(255)|1||0
3|role|VARCHAR(255)|1||0
4|profile_image_url|VARCHAR(255)|1||0
5|timestamp|DATE|1||0
```

My scenario is that on proxmox, I was migrating from one machine to another. I never put into account that /var/lib/docker/volumes/ollama-webui/_data/ollama.db is not part of the cluster or the virtual environment in pve, so that I only did rsync.

I believe we should have a migration docs because man, this repo is improving at lightning speed!

And it should include migrating the volume too.

Another question I have is the structure of the table. The user table doesn't seem to have a password. Accessing the sqlite db should also be documented. This not only makes the system much more user-friendly to devs, but also leave plenty of room for customization and improvement for wishlist and experimentation for a probable PR.

**Comment by justinh-rahb** [comment]
Perhaps a command that could be run on the container shell to reset the first admin account password as well? Deleting the whole database isn't exactly an ideal solution to a lost password if you've got other users, and chat histories you'd like to keep.

**Comment by tjbck** [comment]
> I believe we should have a migration docs because man, this repo is improving at lightning speed!

FYI, the db does not need any migration, and will continue to work after you pull the latest image.

**Comment by tjbck** [comment]
If you'd like to start from scratch, you could always just delete the ollama.db file and you'll be able to create a new admin account. As for the password update, you can do so from the admin panel page but I presume you don't have access to it at the moment.

**Comment by oliverbob** [comment]
> Perhaps a command that could be run on the container shell to reset the first admin account password as well? Deleting the whole database isn't exactly an ideal solution to a lost password if you've got other users, and chat histories you'd like to keep.

Exactly my point.

> If you'd like to start from scratch, you could always just delete the ollama.db file and you'll be able to create a new admin account. As for the password update, you can do so from the admin panel page but I presume you don't have access to it at the moment.

Scenario: I think it would be difficult to explain the reason for doing so to a growing number of users to register a second time, whenever an admin is locked out as in my case (due to migration or working from one computer to another) while experimenting with this great UI.

I was just actually doing a scaling mechanism, trying to cluster my servers. The reason for clustering is that Ollama models are getting larger and larger and I was trying to replicate a fail-over, when suddenly I was locked from admin account. I was treated as a user.

My guess is that there was one letter in my address or password that was a mistake. But this can essentially account for a practical human error which rarely happens.

**Comment by tiredcisadmin** [comment]
I need to know how to update my password in the database because the recent name change with the docker compose setup is telling my password is wrong.

**Reply by justinh-rahb** [reply]
You've likely created a new docker volume. Use the old docker volume, your data will still be intact.

Use ✅:
```
-v ollama-webui:/app/backend/data
```

Not 🚫:
```
-v open-webui:/app/backend/data
```

**Reply by tiredcisadmin** [reply]
This is what I did, and my stuff came back. Anyone that's been using the Docker Compose setup since before the name switch should follow this, i.e. do not merge the change to the volume label when you merge from origin/main from Github.

**Comment by Pierrepaslier** [comment]
Hi, I'm trying to delete the admin user I created to be able to disable auth altogether. I've tried to uninstall everything but the initial user I created still shows up and prevents me from disabling sign in. Someone suggested deleting ollama.db but I can't find this file anywhere. Any help would be greatly appreciated.

**Reply by talgolan** [reply]
I, too, had the very same issue @Pierrepaslier. It turns out that ollama.db has been renamed to webui.db.

https://docs.openwebui.com/getting-started/env-configuration/#database_url

For me, the file is located at ../open-webui/backend/data/webui.db.

Delete ../open-webui/backend/data/webui.db and restart setting WEBUI_AUTH=false.

**Comment by JohnDWorld** [comment]
Hi, 

I had the same problem and just delete the volume created for OpenWebUI in docker desktop.

**Comment by destratified** [comment]
if using docker on cli with docker compose, you want to stop docker stack...

`docker compose down`

once containers are stopped...

`docker volume ls`
`docker volume rm docker_open-webui` #or whatever you may have named it

restart stack

`docker compose up -d`

viola
